crates/iota-sdk-ffi/src/uniffi_helpers.rs

```diff
@@ -63,8 +65,6 @@ macro_rules! define_paged_object {
     };
 }
 
-define_paged_object!(SignedTransactionPage, SignedTransaction);
-define_paged_object!(TransactionDataEffectsPage, TransactionDataEffects);
```

Comment:
Can you briefly explained why it's moved from object to record?

Reply:
Because they don't need associated methods, and I found that they made more sense as records